### PR TITLE
DBAAS1-82: Database Details - Cluster Configs

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

This PR adds the breadcrumb to the Database Details page and the Cluster Configs section to the Summary tab.

2022-01-05 18 44 20 localhost 40e930f8cdba

## How to test

Turn on MSW and go to any Database Details page.

### CODE REVIEW

**Review by DevDW — COMMENTED** [review]
This is looking pretty good so far, just a few comments/questions:

- Is the letter spacing tighter compared to the mockups?

- The alignment of the text gets a bit uneven in mobile view:
Screen Shot 2022-01-05 at 10 56 57 AM

**Review by DevDW — APPROVED** [review]
🚢

File: packages/manager/src/factories/databases.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export const databaseFactory = Factory.Sync.makeFactory<Database>({
   region: 'us-east',
   status: pickRandom(possibleStatuses),
   type: databaseTypeFactory.build().id,
+  version: '5.8.13',
```
Comment (bnussman) [inline packages/manager/src/factories/databases.ts:74]: I've also seen `mysql/5.8.13` for the version value, but can address this later.

**Review by bnussman — APPROVED** [review]

File: packages/manager/src/features/Databases/DatabaseDetail/DatabaseSummary/DatabaseSummaryClusterConfiguration.tsx [hunk 2]
```diff
@@ -1,18 +1,117 @@
...
+
+export const databaseStatusMap: Record<DatabaseStatus, Status> = {
+  creating: 'other',
+  running: 'active',
+  failed: 'error',
+  degraded: 'inactive',
+  updating: 'other',
+};
+
+export const databaseEngineMap: Record<Engine, string> = {
```
Comment (bcmertz) [inline packages/manager/src/features/Databases/DatabaseDetail/DatabaseSummary/DatabaseSummaryClusterConfiguration.tsx:44]: This appears to be duplicated with https://github.com/linode/manager/blob/94eeab5cd5fbfaaaee06ed13930160c380ceae28/packages/manager/src/features/Databases/DatabaseLanding/DatabaseRow.tsx#L29-L34

**Review by bcmertz — APPROVED** [review]
LGTM! Might just want to deduplicate databaseEngineMap before merging

**Pull request merged**